sky/packages/updater/lib/main.dart

```diff
@@ -54,27 +98,25 @@ class UpdateTask {
       print('Update skipped. No new version.');
       return;
     }
-    MojoResult result = await _fetchBundle();
-    if (!result.isOk) {
-      print('Update failed while fetching new skyx bundle.');
-      return;
-    }
+    await _fetchBundle();
+    await _validateBundle();
     await _replaceBundle();
     print('Update success.');
   }
 
-  yaml.YamlMap _currentManifest;
+  Map _currentManifest;
```

Comment:
Yes please. My goal is to eventually turn on an analyzer flag that complains about any implicit use of 'dynamic' at all. (Such a flag doesn't exist yet and won't exist until after we get generic methods, but the more we fix now, the less work it'll be later.) "dynamic" ends up being highly unintuitive in places that you wouldn't expect, e.g. `List<int> is List<dynamic>` and `List<dynamic> is List<int>` are both true.